src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -338,15 +338,17 @@ public static boolean isAllEmpty(final CharSequence... css) {
      * @since 3.0 Changed signature from isBlank(String) to isBlank(CharSequence)
      */
     public static boolean isBlank(final CharSequence cs) {
-        int strLen;
```

Comment:
I think that name strLen is more informative

Reply:
Maybe, however 'strLen' sounds like something unchangeable, since String is immutable in Java. I've chosen this variable name to be the same to name of the similar 'sz' variable in isWhitespace() method. The 'sz' name stands for "size", I think. If we decide to change it we should change names of they both.

Reply:
I tend to agree with sz being more appropriate of a name here than strLen. It would appear that no other variable named `strLen` ever deviates from its original assignment (even the non-final ones.) 

Saving a loop variable isn't a great gain in memory usage but I suppose this is a bit more appropriate. Though, thinking ahead for the future - can one imagine having to reference the string length for a future change? That would be my opposition for this.

Reply:
@MarkDacek, this and also the `isWhitespace()` methods are just string scan methods. They don't need the string length except the very beginning. Thus the original value returned by `cs.length()` doesn't need to be memorized.

Reply:
So maybe instead of "sz" the name should be "i" or "index"? It's quite good name for a looping-control variable, isn't?

Reply:
Or `idx`? But aren't we spend too much time for a discussion of one variable name? :-)

Reply:
Hm, I've given another look through and it would appear that other uses of `sz` are final as well. 

I don't object to this, but I would question the need for it as the memory gain isn't noticeable.

Reply:
Regarding the final keyword for ‘sz’, in my view makes sense to make the code uniform and in general, the final keyword is used in the surrounding affected for variables non-mutating (even for variables inside methods).  Personally, I like it because think it makes the code more readable providing information about the mutability of variable

Reply:
So what name do you think is better in this case? `index`, `idx` or something else?

Reply:
idx is fine for me